cfme/control/explorer/policies.py

```diff
@@ -421,6 +422,14 @@ def assign_actions_to_event(self, event, actions):
         view.flash.assert_message('Actions for Policy Event "{}" were saved'.format(
             event))
 
+    @property
+    def exists(self):
+        policies = cfmedb()["miq_policies"]
```

Comment:
This should now use the appliance objects db ```self.appliance.db``` as doing it the other way may get the wrong db object